eventarc/README.md

```diff
@@ -1,20 +1,18 @@
-# Cloud Eventarc Java Samples
+# Eventarc Java Samples
 
 [![Open in Cloud Shell][shell_img]][shell_link]
 
 [shell_img]: http://gstatic.com/cloudssh/images/open-btn.png
 [shell_link]: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/java-docs-samples&page=editor&open_in_editor=blog/README.md
 
-This directory contains samples for Cloud Eventarc.
+This directory contains samples for Eventarc.
```

Comment:
nit: this description and the ones in the table aren't very descriptive if you don't know what Eventarc is.